configs/default/TMTR-TMOTORF7.config

```diff
@@ -104,4 +104,4 @@ set dashboard_i2c_bus = 2
 set flash_spi_bus = 3
 set gyro_1_bustype = SPI
 set gyro_1_spibus = 1
-set pinio_box = 39,-1,-1,-1
+set pinio_box = 40,255,255,255
```

Comment:
@DieHertz: This will require the documentation that TMotor provides for this board to be updated. Can you please confirm that you'll be looking into this?